.github/workflows/terraform-apply.yml

```diff
@@ -0,0 +1,35 @@
+name: Terraform Apply
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'terraform/services/github-actions/**' # Specify the path to trigger the workflow
```

Comment:
We need to move terraform/services/github-actions/deploy-roles to terraform/services/deploy-roles because changes there should not trigger this workflow. We could add `!terraform/services/github-action/deploy-roles` to paths but I would rather flatten the terraform services hierarchy.